### DISCUSSION TITLE
Symlink to current installation

### DISCUSSION THREAD

**Discussion by creydr** [Homebrew/discussions#4947] [Linux] [answered]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.20
ORIGIN: https://github.com/Homebrew/brew
HEAD: d3d51d3448966b5250b5c60110c945040054803d
Last commit: 3 weeks ago
Core tap JSON: 13 Nov 15:23 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :1
HOMEBREW_EDITOR: /usr/bin/nano
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 2.6.10 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: dodeca-core 64-bit unknown_0x6_0xa5
Clang: N/A
Git: 2.43.0 => /bin/git
Curl: 8.2.1 => /bin/curl
Kernel: Linux 6.5.12-300.fc39.x86_64 x86_64 GNU/Linux
OS: Fedora release 39 (Thirty Nine)
Host glibc: 2.38
/usr/bin/gcc: 13.2.1
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: 13.2.0
gcc: 13.2.0
xorg: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  gcc@5
  isl@0.18
  openssl@1.1
```

### Description of issue

I am looking for a way/symlink which points always to the latest installed version directory? 
E.g. for golang it is `/usr/local/Cellar/go/<version>`. In [#8699](https://github.com/Homebrew/brew/issues/8699) it was mentioned, that `/usr/local/opt/` should do this, but I can't find it on my system & the issue/answer was labeled as `outdated`.
(my problem is actually the same as described in [#8699](https://github.com/Homebrew/brew/issues/8699))

**Comment by SMillerDev** [comment, marked as answer]
You're on Linux so the Cellar path is: `/home/linuxbrew/.linuxbrew/Cellar/go/<version>`
and the opt path is: `/home/linuxbrew/.linuxbrew/opt/go`

**Reply by creydr** [reply]
Awesome. Thanks a lot :+1:
